src/ui/components/button/Button.module.css

```diff
@@ -0,0 +1,16 @@
+.button {
+  padding: 10px 16px;
+  font-size: 14px;
+  line-height: 17px;
+  background-color: rgb(255, 153, 51);
+  border: 1px solid rgb(255, 128, 0);
+  font-weight: 600;
+  color: white;
+  border-radius: 6px;
+  font-family: 'Rubik';
```

Comment:
Its easier to just apply Rubik to the body itself, then individually handle cases that are not rubik

Reply:
Updated